src/main/java/am2/blocks/tileentities/TileEntityCraftingAltar.java

```diff
@@ -763,7 +763,7 @@ private int getInstability() {
 			}
 		}
 
-		return (int)instability;
+		return (int) (instability > 1.0F ? instability : 1.0F);

```

Comment:
This is incorrect understanding of the stability calculations. The number must be larger than 1 if instability is to exist at all. Thus the "> 1.0F" check at the return would disable instability altogether.